packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -116,6 +116,7 @@ class Scrollable extends StatefulWidget {
     this.restorationId,
     this.scrollBehavior,
     this.clipBehavior = Clip.hardEdge,
+    this.copyInterceptor,
```

Comment:
This is not ideal, we will have to drill this down for every scrollable subclass or scrollview.

Reply:
Do you have any examples? Since `MultiSelectableSelectionContainerDelegate` forces you to implement the `copyInterceptor` parameter every current implementation of `MultiSelectableSelectionContainerDelegate` now has this parameter.

Reply:
I meant you also have CustomScrollView, ListView, PageView, TabView, GridView...etc are all using Scrollable. if we do it this way, we have to add this copyInterceptor to all of them. and if we add more parameter, we will add to all of them as well.

I also not a fan of adding selection related API to Widgets. it will be confusing for people who do not use SelectionArea.